workflows/phageannotator/main.nf

```diff
@@ -71,38 +42,21 @@ def multiqc_report = []
 
 workflow PHAGEANNOTATOR {
 
+    take:
+    fastq_gz    // [ [ meta ], reads.fastq.gz ]     , reads (mandatory)
+    fasta_gz    // [ [ meta ], assembly.fastq.gz ]  , assemblies/genomes (mandatory)
```

Comment:
I'm not sure how these are intended to be used, but if they're supposed to be a fasta linked to a sample it all needs to be in one channel.

Reply:
The fastq_gz channel contains reads, and the fasta_gz channel contains the assemblies (both from the same sample). Do you think I should make a sample that contains the reads and assemblies?